flutter_vlc_player/lib/src/vlc_player_controller.dart

```diff
@@ -231,7 +231,7 @@ class VlcPlayerController extends ValueNotifier<VlcPlayerValue> {
             activeSpuTrack: event.activeSpuTrack,
             isPlaying: event.isPlaying,
             playingState:
-                event.isPlaying ? PlayingState.playing : value.playingState,
+                event.isPlaying! ? PlayingState.playing : value.playingState,
```

Comment:
Can we be absolutely sure we're not getting a null here?

Reply:
resolved